airbyte-cdk/java/airbyte-cdk/dependencies/build.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
     api 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
     api 'com.google.guava:guava:33.0.0-jre'
     api 'commons-io:commons-io:2.15.1'
-    api ('io.airbyte.airbyte-protocol:protocol-models:0.7.0') { exclude group: 'com.google.api-client', module: 'google-api-client' }
+    api ('io.airbyte.airbyte-protocol:protocol-models:0.9.0') { exclude group: 'com.google.api-client', module: 'google-api-client' }
```

Comment:
I haven't looked at this closely before... why does out CDK have `com.google.api-client` as a sub-dependency?

Reply:
I'm also not sure. Is this something that the destination team needs for bigquery? cc : @stephane-airbyte